test/jest.setup.ts

```diff
@@ -1,17 +0,0 @@
-// Global setup is performed before a TypeScript environment is made available, so we need to create the environment manually
-require('ts-node/register')
-
-import fs from 'fs'
-import path from 'path'
-
-const dir = path.join(__dirname, '..', '.test-data')
-
-const setup = async () => {
-  if (fs.existsSync(dir)) {
-    fs.rmdirSync(dir, { recursive: true })
-  }
-
-  fs.mkdirSync(dir, 0o755)
-}
-
-export default setup
```

Comment:
I found it weird that global setups and teardowns depended on each other, so I removed them for the unit tests and just copy pasted the code in the others